source/cps/index.rst

```diff
@@ -0,0 +1,33 @@
+CPS
+===
+
+:软件名称: CPS
+:主页: http://www.eas.slu.edu/eqc/eqccps.html
+:源码地址: http://www.eas.slu.edu/eqc/eqc_cps/CPS/cpslisc.html
+:官方文档: http://www.eas.slu.edu/eqc/eqc_cps/CPS/CPS330.html
+:作者: `Saint Louis University Earthquake Center <http://www.eas.slu.edu/eqc/eqc.html>`__
```

Comment:
Is the author correct?

Reply:
I am not sure. But I think Robert is not the only author, because Charles. J. Ammon is also one author in the documentation.

Reply:
Perhaps we should list them two.

Reply:
I quickly check Herrmann (2013) SRL.

The first version of CPS (Herrmann, 1978) was developed by Herrmann between 1971 and 1974 as part of his Ph.D. studies at Saint Louis University.

Maybe we can only list Robert?

Reply:
Maybe Hermann only, as he is the only author of the SRL paper, right?

Reply:
Yes, he is the only author of (2013, SRL) and (1978).

Reply:
Only list Robert now.